app/Main.hs

```diff
@@ -104,7 +139,12 @@ parseOptions =
 
     files = many (argument str (metavar "DOCKERFILE..." <> action "file"))
 
-    lintOptions = Hadolint.LintOptions <$> ignoreList <*> parseRulesConfig
+    lintOptions = Hadolint.LintOptions <$> errorList
```

Comment:
I would nest this new stuff below an `override` key:

```yaml
override:
    error:
      - DL3002
```

Reply:
Agreed. This sounds reasonable.